pupil-api/src/services/feature.service.ts

```diff
@@ -0,0 +1,52 @@
+import logger from './log.service'
+import config from '../config'
+import * as R from 'ramda'
+import * as featureToggles from 'feature-toggles'
+
+export interface IFeatureService {
+  redisAuthMode (): boolean
+}
+
+let initialised: boolean
+
+export class FeatureService implements IFeatureService {
+
+  constructor () {
+    if (!initialised) this.init()
+  }
+
+  redisAuthMode (): boolean {
+    return featureToggles.isFeatureEnabled('preparedChecksInRedis')
+  }
+
+  private init (): void {
+    /**
+     * Load feature toggles
+     */
+    logger.info('ENVIRONMENT_NAME : ' + config.Environment)
+    const environmentName = config.Environment
+    let featureTogglesSpecific: string
+    let featureTogglesDefault: string
+    let featureTogglesSpecificPath: string
+    let featureTogglesDefaultPath: string
+    try {
+      featureTogglesSpecificPath = '../feature-toggles/feature-toggles.' + environmentName
```

Comment:
shouldn't it throw in case `config.Environment` is not set?

Reply:
no, because it defaults to `Local-Dev` if not set

Reply:
ok fair enough